chainer_chemistry/models/prediction/padding_model_wrapper.py

```diff
@@ -0,0 +1,13 @@
+import chainer
```

Comment:
sorry could you define this class in example? this is temporal class.

Reply:
fixed!